tests/cbmc/proofs/s2n_dh_compute_shared_secret_as_client/Makefile

```diff
@@ -39,7 +39,7 @@ PROJECT_SOURCES += $(SRCDIR)/utils/s2n_mem.c
 PROJECT_SOURCES += $(SRCDIR)/utils/s2n_result.c
 PROJECT_SOURCES += $(SRCDIR)/utils/s2n_safety.c
 
-UNWINDSET += s2n_stuffer_write_network_order.4:9
+UNWINDSET += s2n_stuffer_write_network_order.10:9
```

Comment:
What's the significance of 4 and 10?

Reply:
Those extra guards are macros that each expand to multiple (`do { ... } while(0)`) loops. These additional loops affect the loop counts used by CBMC, which may need those unwinding hints when it cannot by itself infer loop bounds. When this value is incorrect, loop-unwinding-assertion failures will be observed, as has happened here in earlier runs.

Reply:
Thanks!